start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
added bash in dockerfile, but somehow not working on my local, used sh shell in this script and working